### PR TITLE
Lesson 20

### PR DESCRIPTION
**Pull request by Stephs15**

### CODE REVIEW

**Review by guicheffer — APPROVED** [review]
That's really nice :)

File: src/lessons/20_security/security.slides.html [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+        <section>
+            <h3>The callstack can only ever run one thing at a time</h3>
+        </section>
+        <section>
+            <h1>Security!</h1>
+            <img src="./images/hackerman.gif" alt="hackerman gif">
+        </section>
+        <section>
+          <h2>Security</h2>
+          <p class="fragment">Javascript is public!</p>
```
Comment (oktupol) [inline src/lessons/20_security/security.slides.html]: I'd rephrase this. Maybe "Javascript is a client-side language" or "Javascript code can be inspected by users"

Comment (Stephs15) [reply src/lessons/20_security/security.slides.html]: fixed!

**Review by oktupol — APPROVED** [review]
